operator/src/main/resources/instances/developer.properties

```diff
@@ -26,5 +26,6 @@ storage.min-margin=1Gi
 # this is arbitrary, we don't expect request for multiple developer units
 kafka.partition-capacity=1500
 kafka.message-max-bytes=1048588
+kafka.topic-config-enforced-rule=flush.ms:9223372036854775807,index.interval.bytes:4096,compression.type:producer,flush.messages:9223372036854775807,min.cleanable.dirty.ratio:0.5,file.delete.delay.ms:60000,segment.index.bytes:10485760,preallocate:false,unclean.leader.election.enable:false,min.insync.replicas:1
```

Comment:
It seems a little off that the plugin needs the value of `min.insinc.replicas` specified twice. What if the `:1`/value is optional here and the plugin interprets that as "use the config value of `min.insync.replicas` as the enforcement value instead".

Reply:
>  What if the `:1`/value is optional here and the plugin interprets that as "use the config value of `min.insync.replicas` as the enforcement value instead".

Sounds good to me. Thanks.
@k-wall , if you have no objection, I'll go with it.

Reply:
I made an alternative suggestion to avoid the duplication.